packages/ember-toucan-core/src/components/form/controls/character-count.hbs

```diff
@@ -0,0 +1,3 @@
+<span class="type-xs-tight" ...attributes>{{this.currentCount}}
```

Comment:
Instead of another wrapping div over on https://github.com/CrowdStrike/ember-toucan-core/pull/115/files#diff-1f29edf2e3cdd8fbd4bbbdf9d06e2a675cbc42ffab5bc1a77b371214c86a5b46R65 for the yielded block, could we apply those styles here instead and only yield back the block?

The block:

```hbs
{{yield (hash id=field.id) to="character"}}
```

And then update this:

```hbs
<span class="type-xs-tight text-body-and-labels flex-1 text-right" ...attributes>{{this.currentCount}} / {{@max}}</span>
```

Since we are opinionated on how our character counter looks, I think it's safe to add some more layout classes here probably? I'm just wondering if we can cut down on some of the DOM elements.  If not, that's fine!

Reply:
I removed it from this `character-count` component and added it to the demo.